Server/src/Server.Infrastructure/Services/SecurityTokenService.cs

```diff
@@ -39,7 +38,7 @@ public SecurityTokenService(AuthSettings authSettings)
         _authSettings = authSettings;
     }
 
-    public string GenerateAccessToken(Guid identityId, string email, ICollection<string> roles)
+    public string GenerateAccessToken(Guid identityId, string email, Guid? libraryId, ICollection<string> roles)
```

Comment:
w tym momencie chyba powinniśmy przyjmować w tej metodzie tylko Identity, bo zaraz dojdą uprawnienia i wgl

Reply:
W sensie żeby to libraryId to już pobrać z Identity?